lib/virt_autotest/kernel.pm

```diff
@@ -34,23 +34,31 @@ use version_utils;
 our @EXPORT = qw(check_virt_kernel);
 
 sub check_virt_kernel {
-    my $guest  = shift // '';
-    my $suffix = shift // '';
-    my $go_to_target = $guest eq '' ? '' : "ssh root\@$guest";
+    my $guest      = shift // '';
+    my $suffix     = shift // 'localhost';
+    my $kernel_log = shift // '/tmp/virt_kernel.txt';
+    my $go_to_target = $guest eq 'localhost' ? '' : "ssh root\@$guest";
```

Comment:
This looks like a more complex subroutine with multiple parameters. What do you think about changing this to named parameters, like

    check_virt_kernel(guest => "SLES15SP1", go_to_target => "localhost")

Especially for subroutines that we intend to re-use I think this might increase the readability in our tests